packages/btcindexer/src/btcindexer.ts

```diff
@@ -460,6 +486,53 @@ export class Indexer {
 		}
 	}
 
+	async detectMintedReorgs(): Promise<void> {
+		logger.debug({ msg: "Cron: Checking for reorgs on minted transactions" });
+
+		const mintedTxs = await this.storage.getMintedTxs();
```

Comment:
1. I think this should be called when we process blocks (in the queue handler)
2. You want to query all minted TXs? This is not good. This should be based on the block we are inserting (so all minted txs  in block height >= processingBlock.height WHEN we detected a reorg - so you must check about the Reorg)